modules/installation-bare-metal-dns-record-type.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/upi/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/upi/installing-bare-metal.adoc
+// * installing/installing_bare_metal/upi/installing-restricted-networks-bare-metal.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-bare-metal-dns-record-type_{context}"]
+= Configuring the DnsRecordsType parameter
+
+[role="_abstract"]
+You can use the `DnsRecordsType` parameter in your `infrastructure.config.openshift.io` custom resource (CR) to set if the internal DNS service or an external source provides the necessary records for `api`, `api-int`, and `ingress` DNS records.
+
+:FeatureName: Configuring the DnsRecordsType parameter
+include::snippets/technology-preview.adoc[]
+
+To use the parameter, you must enable the `OnPremDNSRecords` feature gate in the `config.yaml` file.
```

Comment:
The featuregate is part of the TechPreview featureset. There's no explicit enablement required.